src/main/java/optic_fusion1/mcantimalware/configuration/ConfigurationOptions.java

```diff
@@ -46,12 +46,9 @@ public char pathSeparator() {
      * only in how you access the data. The default value is '.'.
      *
      * @param value Path separator
-     * @return This object, for chaining
      */
-    @NotNull
-    public ConfigurationOptions pathSeparator(char value) {
+    public void pathSeparator(char value) {

```

Comment:
Every change to this class is useless, it breaks the Builder Pattern it had previously. I.e.
FileConfiguration config;
config.options().copyDefaults(true).copyHeader(true);
is no longer possible because of these changes (I checked)